src/client/minimap.cpp

```diff
@@ -683,8 +681,17 @@ MinimapMarker *Minimap::addMarker(scene::ISceneNode *parent_node)
 
 void Minimap::removeMarker(MinimapMarker **m)
 {
-	m_markers.remove_if([ptr = *m](const auto &up) { return up.get() == ptr; });
+	MinimapMarker *ptr = *m;
 	*m = nullptr;
+	for (auto it = m_markers.begin(); it != m_markers.end(); ++it) {
```

Comment:
Might it be helpful to move this to a separate utility function? e.g. `unordered_vector_erase(vector &, value)`

Reply:
maybe, but it will end up more LoC than just doing it inline